src/vs/workbench/contrib/chat/browser/chatAttachmentModel.ts

```diff
@@ -57,6 +59,21 @@ export class ChatAttachmentModel extends Disposable {
 		this._onDidChangeContext.fire();
 	}
 
+	async attachScreenshot(): Promise<void> {
+		this.clear();
+		const imageData = await generateFocusedWindowScreenshot();
+		if (!imageData) {
+			return;
+		}
+		this.addContext({
+			id: 'screenshot-focused-window',
+			name: localize('screenshot', 'Screenshot'),
+			value: imageData,
+			isImage: true,
+			isDynamic: true
+		});
```

Comment:
A helper to make this would be nice to reduce repetition across the 2 files and chance that they accidentally diverge.